packages/playwright/src/runner/testServer.ts

```diff
@@ -324,6 +323,10 @@ export class TestServerDispatcher implements TestServerInterface {
       process.env.PW_LIVE_TRACE_STACKS = '1';
     else
       process.env.PW_LIVE_TRACE_STACKS = undefined;
+    if (params.pageSnapshot)
```

Comment:
We don't need this, one can run a server with `PW_TEST_PAGE_SNAPSHOT` env instead.